examples/simple-router-wasm/src/main.rs

```diff
@@ -0,0 +1,51 @@
+//! Run with
+//!
+//! ```not_rust
+//! cd examples && cargo run -p example-simple-router-wasm
```

Comment:
But this doesn't compile to Wasm. What about adding `--target wasm32-unknown-unknown` and running it in a Wasm runtime?

Reply:
The goal here is to have an example that runs on the user's local machine. We could add a wasm runtime, but then we'd need to have to determine a runtime, add a runtime specific interface, get the user to install the runtime, and then support that runtime in CI. That feels like much more of a maintenance burden. I think it's better to provide some guidance and let the user figure it out.

Reply:
I agree and good that we call this out in the example.